test/test_rospy/test/rostest/test_rospy_client_online.py

```diff
@@ -68,141 +68,176 @@ def __init__(self, *args):
         rospy.init_node('test_rospy_online')
 
     def test_log(self):
-        # we can't do anything fancy here like capture stdout as rospy
-        # grabs the streams before we do. Instead, just make sure the
-        # routines don't crash.
-        
-        real_stdout = sys.stdout
-        real_stderr = sys.stderr
 
+        import logging
+        import rosgraph.roslogging
+
+        rosout_logger = logging.getLogger('rosout')
+
+        print("HANDLERS : " + str(rosout_logger.handlers))
+        self.assertTrue(len(rosout_logger.handlers) == 2)
+        self.assertTrue(rosout_logger.handlers[0], rosgraph.roslogging.RosStreamHandler)
+        # self.assertTrue(rosout_logger.handlers[1], rospy.impl.rosout.RosOutHandler)
```

Comment:
I left it there to illustrate what the handlers should be. I might as well actually check it.